application/lib/utils/styles.dart

```diff
@@ -41,9 +41,48 @@ class Styles {
     color: MyColors.textColor,
   );
 
+  static const TextStyle forgotPasswordStyles = TextStyle(
+    fontFamily: robotoFont,
+    fontSize: 13,
+    color: Colors.grey,
+  );
+
   static const TextStyle eventTitleStyles = TextStyle(
     fontFamily: robotoFont,
-    fontSize: 20,
+    fontSize: 30,
+    fontWeight: FontWeight.bold,
+    color: MyColors.darkBlueColor,
+  );
+
+  static const TextStyle menuTextStyles = TextStyle(
```

Comment:
Itt véletlen bekerültek olyan stílusok, amik még nincsenek is használva. Azokat kérlek szedjük ki.